mijia/README.md

```diff
@@ -3,9 +3,51 @@
 [![crates.io page](https://img.shields.io/crates/v/mijia.svg)](https://crates.io/crates/mijia)
 [![docs.rs page](https://docs.rs/mijia/badge.svg)](https://docs.rs/mijia)
 
-`mijia` is a library for connecting to Xiaomi Mijia 2 Bluetooth temperature/humidity sensors.
+A library for connecting to Xiaomi Mijia 2 Bluetooth temperature/humidity sensors.
 
-See the [examples](examples/) directory for examples of how to use it.
+Currently only supports running on Linux, as it depends on BlueZ for Bluetooth.
+
+## Usage
+
+```rust
+// Create a new session. This establishes the D-Bus connection. In this case we ignore the join
+// handle, as we don't intend to run indefinitely.
+let (_, session) = MijiaSession::new().await?;
```

Comment:
bonus points if this can be tested using something like https://github.com/budziq/rust-skeptic :D

Reply:
(skeptic itself recommends using https://crates.io/crates/doc-comment for this use-case)

Reply:
Unfortunately I can't get `doc-comment` to work, because it doesn't seem to have any way to add the surrounding boilerplate to let `.await` and `?` work in the example. `rust-skeptic` might work, but requires adding a `build.rs` file which I'd prefer not to.